libs/platform/ebpf_epoch.c

```diff
@@ -64,6 +64,9 @@
 typedef struct _ebpf_epoch_state
 {
     int64_t epoch; // The highest epoch seen by this epoch state.
+#if defined(_DEBUG)
```

Comment:
Can you merge the is_preemtible into the upper bits? This change will alter the number of entries in the epoch thread table between debug and release, which may alter testing.